content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -255,3 +255,66 @@ spec:
     tls:
       mode: DISABLE
 {{< /text >}}
+
+## Migrating the `RbacConfig` to `ClusterRbacConfig`
+
+The `RbacConfig` is deprecated due to a [bug](https://github.com/istio/istio/issues/8825). You should
+migrate to `ClusterRbacConfig` if you are currently using `RbacConfig`. The bug reduces the scope of
+the object to be namespace-scoped in some cases. The `ClusterRbacConfig` follows the exact same
+specification as the `RbacConfig` but with correct cluster scope implementation.
+
+To migrate the `RbacConfig` to `ClusterRbacConfig`:
+
+1. We provide the `convert_RbacConfig_to_ClusterRbacConfig.sh` script to automate the migration,
+   depending on if you have the [Istio installation package](/docs/setup/kubernetes/download-release):
+
+    - If you have the Istio installation package
+
+        Run the script with the following command under the directory of Istio installation package:
+
+        {{< text bash >}}
+        $ ./tools/convert_RbacConfig_to_ClusterRbacConfig.sh
+        {{< /text >}}
+
+    - If you don't have the Istio installation package
+
+        Download and run the script directly with the following command:
+
+        {{< text bash >}}
+        $ curl -L https://raw.githubusercontent.com/istio/istio/master/tools/convert_RbacConfig_to_ClusterRbacConfig.sh | sh -
+        {{< /text >}}
+
+The script automates the following operations:
+
+1. Create a `ClusterRbacConfig` with same specification as the existing `RbacConfig`. This
```

Comment:
See comment on line 318 for the reason behind this edit suggestion. Do not use the value in `kind` to refer to configuration files unless you are specifically referencing the code:
1. The script creates a cluster RBAC configuration file with same specification as the existing RBAC configuration file because